test/tests/interfaces/pbs_preempt_params.py

```diff
@@ -48,8 +48,10 @@ class TestPreemptParamsQmgr(TestInterfaces):
     error_unauth = "Unauthorized Request"
```

Comment:
Not used any where now , do we need to removed it ?

Reply:
I have added the logic that uses it, it was removed by mistake.